source/_integrations/legrand_homepluscontrol.markdown

```diff
@@ -0,0 +1,103 @@
+
+As part of the authentication exchange, a callback request is made to the REDIRECT_URL that was entered when you registered the application with the *Works with Legrand* platform. This redirect URL will take on the default value of your Home Assistant instance URL, but because this could differ from the URL that you entered in the Legrand registration form, the integration allows you to explicitly specify the redirect URL parameter that you would like to use.
+
+## Polling Interval Configuration
+
+The Legrand `Home+ Control` API is a cloud-based polling interface. This Home Assistant integration presents 3 different polling intervals:
+1) Plant information request
+2) Plant topology request
+3) Module status request
+
+Refer to the [nomenclature](#api-nomenclature) section for details of what these names mean.
+Refer to the [API limitations](#api-limitations) section to better understand the reason for these polling intervals.
+
+### Plant Information Request
+
+This call retrieves the information of the plants that are associated to a user. It is assumed that this information does not change very often. For the most part, a user will have a single *Legrand Home+ Control* gateway in their home, i.e. a single *plant*, and this will not change frequently.
+
+For this reason, the default polling interval for the plant information is set to a large value: 7200 seconds (2 hours).
+
+### Plant Topology Request
+
+This call retrieves the *layout* of the devices that make up a *plant*. This provide the list of device, their names and some additional information of the devices (but not their status).
+
+It is assumed that plant topology is not changing very often, so the default polling interval for this information is set to a default value of 3600 seconds (1 hour).
+
+### Module Status Request
+
+This call retrieves the current operating status of the devices that make up the plant's topology - this includes the *reachability* of the device and its on/off status. 
+
+This information is expected to change frequently, so the polling interval is set to a lower default value of 300 seconds (5 minutes). This is still not as frequent as would be ideal, but is still reasonable considering the API calls-per-day limitation.
+
+
+## API Nomenclature
+
+Within the context of the `Home+ Control` API you may come across the following terms:
+* *Plant*: This is the term used to represent a *home* that holds the Legrand devices. In practice, a *plant* is represented by the *Legrand Home+ Control* gateway that acts as the central hub of the rest of the devices in the home network (uses Zigbee).
+* *Module*: This is the term used to represent a generic device within the *plant*, i.e. a light, a plug, a remote, etc.
+* *Light*: This is the term used to represent a light switch (or a micro-module). It is not modeled as your usual light entity because there are no brightness, color, etc. controls. It is modeled as a simple on/off switch component.
```

Comment:
```suggestion
* *Light*: This is the term used to represent a light switch (or a micro-module). It is not modeled as your usual light entity because there are no brightness, color, etc. controls. It is modeled as an on/off switch.
```